File: src/main/java/game/Controller.java
package game;

public class Controller {
    private Game running;

    private void setRunning(Game running) {
        this.running = running;
    }

    public Game newGame() {
        Game game = new Game();
        setRunning(game);
        return game;
    }

    public void generateMove(Auswahl auswahl){
        int zufallsZahl = (int) (Math.random() * 5);
        Auswahl computer = Auswahl.values()[zufallsZahl];
        game.draft(auswahl, computer);
    }
}

File: src/main/kotlin/game/Controller.kt
package game;

public class Controller {
    private Game running;

    private void setRunning(Game running) {
        this.running = running;
    }

    public Game newGame(int maxRounds, PlayerConfiguration playerConfiguration) {
        Game game = new Game(maxRounds, playerConfiguration);
        setRunning(game);
        return game;
    }

    public Game currentGame() {
        return running;
    }

    public void generateMove(Auswahl auswahl){
        running.advanceDraft(auswahl);
    }
}
